components/automate-ui/src/app/modules/infra-proxy/reset-admin-key/reset-admin-key.component.html

```diff
@@ -4,13 +4,13 @@
       <form [formGroup]="resetKeyForm">
         <chef-form-field>
           <label>
-            <span class="label">New Admin Key<span aria-hidden="true">*</span></span>
+            <span class="label">New Admin Key <span aria-hidden="true">*</span></span>
             <textarea rows="27" cols="10" chefInput placeholder="-----BEGIN RSA PRIVATE KEY -----"
             formControlName="admin_key" data-cy="update-org-admin-key"></textarea> 
           </label>
           <chef-error
             *ngIf="(resetKeyForm.get('admin_key').hasError('required') || resetKeyForm.get('admin_key').hasError('pattern')) && resetKeyForm.get('admin_key').dirty">
-            Admin key is required.
+            New Admin key is required.
```

Comment:
I think we want this:
```suggestion
            New admin key is required.
```